### PR TITLE
Explicitly set request for latency pods

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #22072

### CODE REVIEW

**Comment by wojtek-t** [comment]
@gmarek - this is now ready for review
PTAL

File: test/e2e/density.go [hunk 1]
```diff
@@ -439,6 +458,17 @@ var _ = Describe("Density", func() {
...
 				close(stopCh)
 
+				nodeToLatencyPods := make(map[string]int)
+				for _, item := range latencyPodsStore.List() {
+					pod := item.(*api.Pod)
+					nodeToLatencyPods[pod.Spec.NodeName]++
+				}
+				for node, count := range nodeToLatencyPods {
+					if count > 1 {
+						Logf("%s: %d latency pods scheduled on it", node, count)
```
Comment (gmarek) [inline test/e2e/density.go]: "%d latency pods scheduled on %s", count, node

**Comment by gmarek** [comment]
One nit to make logs easier to spot and compile errors:
```
_output/local/go/src/k8s.io/kubernetes/test/e2e/density.go:443: cannot use 0 (type int) as type "k8s.io/kubernetes/pkg/api/resource".Quantity in assignment
_output/local/go/src/k8s.io/kubernetes/test/e2e/density.go:444: cannot use 0 (type int) as type "k8s.io/kubernetes/pkg/api/resource".Quantity in assignment
```

**Comment by wojtek-t** [comment]
@gmarek - PTAL

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please github issue: #IGNORE (jenkins didn't pick up this PR)

**Pull request merged**